api/test/wfl/system/v1_endpoint_test.clj

```diff
@@ -15,6 +15,7 @@
 
 (def create-wgs-workload (make-create-workload workloads/wgs-workload-request))
 (def create-aou-workload (make-create-workload workloads/aou-workload-request))
+(def create-xx-workload (make-create-workload workloads/xx-workload-request))
 (defn create-copyfile-workload [src dst]
   (endpoints/create-workload (workloads/copyfile-workload-request src dst)))
```

Comment:
Ugh. Doesn't that hurt?

Reply:
Yeah - it's sad. This is so that we can inject a "temporary folder" into the workload. Perhaps we don't need this now we have the dedicated inputs bucket and a temporary output folder